librepo.spec

```diff
@@ -86,7 +86,7 @@ mkdir build-py3
 %build
 %if %{with python3}
 pushd build-py3
-  %cmake -DPYTHON_DESIRED:FILEPATH=%{__python3} %{!?with_zchunk:-DWITH_ZCHUNK=OFF} -DENABLE_PYTHON_TESTS=%{?with_pythontests:ON}%{!?with_pythontests:OFF} ..
```

Comment:
I'm not a packager so syntax like this is unfamiliar to me, where does ```%{__python3}``` come from and why was it provided as a "FILEPATH" when it's supposed to just be a [string containing an integer](https://github.com/rpm-software-management/librepo/pull/219/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aL73)?